conda/core/repodata.py

```diff
@@ -406,6 +407,7 @@ def fetch_repodata(url, schannel, priority,
                    cache_dir=None, use_cache=False, session=None):
     cache_path = join(cache_dir or create_cache_dir(), cache_fn_url(url))
 
+    url = unquote_plus(url) if url else url
```

Comment:
I'm guessing this is ok, but not sure why it's needed here.  Feels better to move this to later.